conf/app.routes

```diff
@@ -53,10 +53,15 @@ POST        /create-profile/ukims-number/check
 GET         /update-record/:recordId/categorisation/start                                   controllers.CategoryGuidanceController.onPageLoad(recordId: String)
 POST        /update-record/:recordId/categorisation/start                                   controllers.CategoryGuidanceController.onSubmit(recordId: String)
 
-GET         /create-record/commodity-code                                                   controllers.CommodityCodeController.onPageLoad(mode: Mode = NormalMode)
-POST        /create-record/commodity-code                                                   controllers.CommodityCodeController.onSubmit(mode: Mode = NormalMode)
-GET         /create-record/commodity-code/check                                             controllers.CommodityCodeController.onPageLoad(mode: Mode = CheckMode)
-POST        /create-record/commodity-code/check                                             controllers.CommodityCodeController.onSubmit(mode: Mode = CheckMode)
+GET         /create-record/commodity-code                                                   controllers.CommodityCodeController.onPageLoadCreate(mode: Mode = NormalMode)
+POST        /create-record/commodity-code                                                   controllers.CommodityCodeController.onSubmitCreate(mode: Mode = NormalMode)
+GET         /create-record/commodity-code/check                                             controllers.CommodityCodeController.onPageLoadCreate(mode: Mode = CheckMode)
+POST        /create-record/commodity-code/check                                             controllers.CommodityCodeController.onSubmitCreate(mode: Mode = CheckMode)
+
+GET         /update-record/:recordId/commodity-code          controllers.CommodityCodeController.onPageLoadUpdate(mode: Mode = NormalMode, recordId: String)
```

Comment:
just the minor one to get the indentation right from `controllers.` to be line with above ones